docs/logging-and-managing-experiment-results/index.rst

```diff
@@ -2,101 +2,153 @@
+
+What you need to add to your code to start logging
+--------------------------------------------------
+[loom-placeholder]
+
+Bare minimum are one import and two methods:
+
+.. code-block:: python3
 
     import neptune
-    import numpy as np
 
-    # select project
-    neptune.init('shared/onboarding',
-                 api_token='ANONYMOUS')
+    # Set project
+    neptune.init('my_workspace/my_project')
 
-    # define parameters
-    PARAMS = {'decay_factor': 0.7,
-              'n_iterations': 117}
+    # Create experiment
+    neptune.create_experiment()
 
-    # create experiment
-    neptune.create_experiment(name='quick_start_example',
-                              params=PARAMS)
+These are usually just copy&paste into your existing project code.
 
-    # log some metrics
-    for i in range(1, PARAMS['n_iterations']):
-        neptune.log_metric('iteration', i)
-        neptune.log_metric('loss', PARAMS['decay_factor']/i**0.5)
-        neptune.log_text('text_info', 'some value {}'.format(0.95*i**2))
+.. note::
 
-    # add tag to the experiment
-    neptune.append_tag('quick_start')
+    Remember to :ref:`create project <create-project>` and :ref:`setup API token <how-to-setup-api-token>` before you create an experiment using snippet above.
 
-    # log some images
-    for j in range(5):
-        array = np.random.rand(10, 10, 3)*255
-        array = np.repeat(array, 30, 0)
-        array = np.repeat(array, 30, 1)
-        neptune.log_image('mosaics', array)
+Now, that the experiment is created you can start logging metrics, losses, images, model weights or whatever you feel relevant to keep track of in your experiment.
 
-.. note:: Save the code as ``main.py`` and run it using the command: ``python main.py``.
+Overall idea for logging is to use methods similar to this:
 
-**More info:**
+.. code-block:: python3
 
-.. toctree::
-   :maxdepth: 1
+    neptune.log_metric('acc', 0.95)
 
-   Logging experiment data <logging-experiment-data/index.rst>
-   Updating existing experiments <updating-existing-experiment/index.rst>
-   Downloading experiment data programmatically <downloading-experiment-data-programmatically/index.rst>
+Generic recipe being:
 
-.. External links
+.. code-block:: python3
 
-.. |Youtube Video| raw:: html
+    neptune.log_X('some_name', some_value)
 
-    <iframe width="720" height="420" src="https://www.youtube.com/embed/w9S5srkfSI4" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+Where ``X`` could be metric, artifact, chart, pickle, etc. Check the :ref:`logging section <what-you-can-log>` for a complete list.
 
-.. |neptune-client-github| raw:: html
+.. note::
 
-    <a href="https://github.com/neptune-ai/neptune-client" target="_blank">Neptune client</a>
+    If you work in Notebooks, you need to place :meth:`~neptune.experiments.Experiment.stop` at the very end of your experiment to make sure that everything will be closed properly.
```

Comment:
it shows as `stop` not as `experiment.stop` or `neptune.stop` and could be confusing (someone can simply add `stop` and wonder what is going on.

Reply:
corrected.